cluster-operator/src/test/java/io/strimzi/operator/cluster/ResourceUtils.java

```diff
@@ -215,20 +215,38 @@ public static ConfigMap createKafkaClusterConfigMap(String clusterCmNamespace, S
         data = new HashMap<>();
```

Comment:
Rather than creating `data`, populating it and ultimately calling `withData()` we should probably use the `SecretBuilder.addToData()` method. There's not much in it, but I think it would be more readable because the creation would be less backwards.

Reply:
I didn't notice the method ... +1